go-controller/pkg/ovn/gateway/gateway.go

```diff
@@ -19,12 +18,6 @@ const (
 	OvnGatewayLoadBalancerIds = "lb_gateway_router"
```

Comment:
do you also want to remove this one? I don't see it being used anywhere unless I am mistaken
Speaking of which I assume the "unused parameter" lint is only scoped to param inside a function... we don't have lints for unused variables in general I guess..

Reply:
If a linter did not drive me to it I prefer not to do it in this PR. A linter won't highlight this one as it's exported so we don't know who might be using it. More than likely it can be removed but it is not connected to any linter added here so if a take up on this then well, why not other things which might make me never finish this PR.

Reply:
that's fine by me just weird the one below got removed but not this one...so if you rebased was wondering

Reply:
the other one was an error, and the error-naming linter drove me to it. In that case the investigation whether it should be renamed or removed made sense.